cephfs/file_test.go

```diff
@@ -415,3 +415,135 @@ func TestFchown(t *testing.T) {
 	err = f2.Fchown(bob, bob)
 	assert.Error(t, err)
 }
+
+func TestFallocate(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+	useMount(t)
```

Comment:
Please make use of `useMount` only in the sub-tests which make use of the mount point (those that call os.Stat).

BTW, after this point I'm going to be requesting the use of the native library stat equivalents now that we have one. Since you possibly started this work before that was merged I'm not going to require it for this PR, but I don't think I want to see it proliferate any more now that we have a call in the library.